pom.xml

```diff
@@ -239,6 +239,7 @@
                 <requireBackwardCompatibility implementation="org.semver.enforcer.RequireBackwardCompatibility">
                   <excludes>
                     <exclude>com/querydsl/core/types/dsl/PathBuilderValidator</exclude>
+                    <exclude>com/querydsl/sql/Union</exclude>
```

Comment:
http://docs.oracle.com/javase/specs/jls/se7/html/jls-13.html Chapter 13, last item of the bulleted list suggests it _should_ be a compatible change.